src/Hydration/IssueProjectHydration.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace DanielBadura\Redmine\Api\Hydration;
+
+use DanielBadura\Redmine\Api\Client;
+use DanielBadura\Redmine\Api\Struct\Issue;
+use ProxyManager\Factory\LazyLoadingValueHolderFactory;
+use ProxyManager\Proxy\LazyLoadingInterface;
+
+/**
+ * @author Marco Giesen <marco.giesen93@gmail.com>
+ */
+class IssueProjectHydration
+{
+    /**
+     * @param Issue $issue
+     * @param Client $client
+     * @return Issue
+     */
+    public function hydrateOneIssue(Issue $issue, Client $client)
```

Comment:
Naming it just `hydrateIssue(..)` should be enough in my opinion.
The other method for multiple issues can you leave like it is.